src/main/java/DeleteCommand.java

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
Maybe can package the command files into a separate package so the `src/main/java` folder has less files